pipeline/metadata/6.0.yaml

```diff
@@ -41,816 +41,816 @@ suites:
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-2
-
-  - name: "test-lc-process-single-bucket"
-    suite: "suites/pacific/rgw/tier-2_rgw-single-bucket-process.yaml"
-    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-2
-
-  - name: "test-rgw-singlesite-to-multisite-tier-2"
-    suite: "suites/pacific/rgw/tier-2_rgw_singlesite_to_multisite.yaml"
-    global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-3
-
-  - name: "test-rgw-quota-management"
-    suite: "suites/pacific/rgw/tier-2_rgw_test-quota-management.yaml"
-    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-3
-
-  - name: "test-rgw-lc-expiration-with-prefix-as-special-character"
-    suite: "suites/pacific/rgw/tier-2_rgw_test-lc-prefix.yaml"
-    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-3
-
-  - name: "tier-2_rgw_test-multisite-metadata-sync"
-    suite: "suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml"
-    global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-3
-
-  - name: "S3Tests against Red Hat RGW with SSL"
-    suite: "suites/quincy/rgw/tier-2_rgw_ssl_s3tests.yaml"
-    global-conf: "conf/pacific/rgw/ec-profile-4+2-cluster.yaml"
-    platform: "rhel-9"
-    rhbuild: "6.0"
-    inventory:
-      openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
-    metadata:
-      - schedule
-      - tier-2
-      - openstack
-      - ibmc
-      - rgw
-      - stage-3
+#  - name: "Testing RADOS basic regression scenarios"
```

Comment:
please create a different commit or this. and its good to remove instead of commenting it out.

Reply:
done